index.html

```diff
@@ -193,6 +197,18 @@ <h2>Implementation Details</h2>
               A fetch.txt file is allowed within the bag. Default: `true`
             </p>
           </li>
+          <li>
+            `Fetch.txt-Required`: `true`|`false`
+            <p>
+              A fetch.txt file is required. Specified when `Allow-Fetch.txt` is `true`. Default is `false`.
```

Comment:
The phrase "A fetch.txt file is required" only describes the situation when `Fetch.txt-Required` is `true`.  Perhaps it would be better to adopt a more neutral description; e.g.,

    Whether or not the `fetch.txt` file is required.

Also, perhaps this could be made slightly stronger by including the word "only"; i.e.,

    Only specify when `Allow-Fetch.txt` is `true`.